lab_1_classify_by_unigrams/main.py

```diff
@@ -64,11 +89,31 @@ def calculate_mse(predicted: list, actual: list) -> float | None:
 
     In case of corrupt input arguments, None is returned
     """
+    if (type(predicted) is not list or type(actual) is not list or len(predicted) == 0 or len(actual) == 0
+            or len(predicted) != len(actual)):
+        return None
+
+    value = 0
+    i = 0
+    while i < len(actual):
+        value += (actual[i] - predicted[i]) ** 2
+        i += 1
+
+    mse = value / i
```

Comment:
return as is without creating the variable

Reply:
done